arrow-libs/core/arrow-core/src/main/kotlin/arrow/core/extensions/either/monad/EitherMonad.kt

```diff
@@ -47,7 +44,10 @@ fun <L, A, B> Kind<Kind<ForEither, L>, A>.flatMap(arg1: Function1<A, Kind<Kind<F
   "EXTENSION_SHADOWED_BY_MEMBER",
   "UNUSED_PARAMETER"
 )
-@Deprecated("@extension kinded projected functions are deprecated", ReplaceWith("Either.tailRecM(arg0, arg1)", "arrow.core.tailRecM"))
+@Deprecated(
+  "@extension kinded projected functions are deprecated",
+  ReplaceWith("Either.tailRecM(arg0, arg1)", "arrow.core.tailRecM")
```

Comment:
These should maybe use the new TailrecM deprecation message instead without replacement to avoid making the user refactor end up in a dead-end.

Reply:
Right, I'll replace all `tailRecM` deprecations with the new message.